src/arch/x86_64/mod.rs

```diff
@@ -80,19 +86,76 @@ pub fn output_message_byte(byte: u8) {
 	unsafe { COM1.send(byte) };
 }
 
+// Right now, the kernel binary has to be hardcoded into the loader.
+// The binary has to be in the same directory (or its whereabouts have to specified in the "include_bytes!" statement).
 #[cfg(target_os = "uefi")]
 pub unsafe fn find_kernel() -> &'static [u8] {
-	&[1, 2, 3]
+	include_bytes!("hermit-rs-template")
 }
```

Comment:
Can we do this as we did with AArch64 before?
Also, this does not have to be `unsafe`.

https://github.com/hermit-os/loader/blob/409e26dee500e5a1ddc38e64fcde7a5918afdd2b/src/arch/aarch64/mod.rs#L82-L86